Baymax/Logger/Logger.swift

```diff
@@ -113,18 +113,18 @@ public struct Logger: TextOutputStream {
             writeString = "\(formatter.string(from: Date())) \(writeString)"
         }
         
-        guard let directory = directory else { return }
+        guard let directory = directory, let writeData = writeString.data(using: .utf8) else { return }
```

Comment:
Similar to the previous comment, it's good we are now doing it in one place but just wanted to confirm it shouldn't be on a background queue? I'm not actually too sure on how long this method could take.